ocs_ci/templates/ocs-deployment/service_account.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: admin
+  namespace: new-namespace
```

Comment:
should we use this namespace?

Reply:
this for template while creating we need to specify namesapce

Reply:
why not change in the template itself and we dont have to do it in code :)

Reply:
namesapce will be generated randomly so just for sample purpose i have specifed that name

Reply:
got it, thanks!